Source/Blazorise/Components/Tabs/TabPanel.razor

```diff
@@ -1,5 +1,15 @@
 ﻿@namespace Blazorise
 @inherits BaseComponent
-<div id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
-    @ChildContent
-</div>
\ No newline at end of file
+
+@if ( Mode == TabsMode.Default )
+{
+    <div id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
+        @ChildContent
+    </div>
+}
+else if ( ( Mode == TabsMode.LazyLoad && Active ) )
+{
+    <div id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
+        @ChildContent
```

Comment:
I would wrap `@ChildContent` with an `if` statement. That way a `div` will still render(without content), but it is a good approach in case someone wished to have a CSS selector targeting a specific child. Or maybe they want to have a tab with a certain size occupying the page.

Reply:
Nice idea.